### PR TITLE
Add more detail to the Compromised TAM section

### PR DESCRIPTION
**Pull request by bremoran**
Merely checking the certificate of a compromised TAM requires reliance on certificate revocation to detect a TAM compromise. This does not deal with APT attacks on TAMs, which are high value targets given the authority they have over TEEs. Additional checks are needed.

### CODE REVIEW

File: draft-ietf-teep-architecture.md [hunk 1]
```diff
@@ -1328,7 +1328,23 @@ may need to be updated.
 ## Compromised TAM
 
 Device TEEs are responsible for validating the supplied TAM certificates
-to determine that the TAM is trustworthy.
+to determine that the TAM is trustworthy. A compromised TAM can install
+arbitrary TAs to, potentially, many devices, which makes TAMs high value
+targets. An advanced, persistent threat actor could potentially compromise
+a TAM without impacting its certificate or raising concern from the TAM's
+operator. To mitigate this threat, a TEEs have several options for
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md]:
```suggestion
operator. To mitigate this threat, TEEs have several options for 
```
grammar

File: draft-ietf-teep-architecture.md [hunk 2]
```diff
@@ -1328,7 +1328,23 @@ may need to be updated.
...
 Device TEEs are responsible for validating the supplied TAM certificates
-to determine that the TAM is trustworthy.
+to determine that the TAM is trustworthy. A compromised TAM can install
+arbitrary TAs to, potentially, many devices, which makes TAMs high value
+targets. An advanced, persistent threat actor could potentially compromise
+a TAM without impacting its certificate or raising concern from the TAM's
+operator. To mitigate this threat, a TEEs have several options for 
+detecting and mitigating a compromised TAM:
+
+1. Apply an ACL to the TAM, indicating which TAs it is permitted to install
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md]:
```suggestion
1. Apply an ACL to the TAM, indicating which TAs it is permitted to install.
```

File: draft-ietf-teep-architecture.md [hunk 3]
```diff
@@ -1328,7 +1328,23 @@ may need to be updated.
...
+2. Use a transparency log to expose a TA compromise: TAMs publish an 
+out-of-band record of TA releases, allowing a TEE to cross-check the TAs
+delivered against the TA releases in order to detect a TAM compromise.
+3. Use remote attestation of the TAM to prove trustworthiness
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md]:
```suggestion
3. Use remote attestation of the TAM to prove trustworthiness.
```

**Review by dthaler — COMMENTED** [review]
Ok except for some editorial nits

**Comment by dthaler** [comment]
Addresses #241

**Review by dthaler — APPROVED** [review]

**Review by mingpeiwk — APPROVED** [review]

**Pull request merged**